app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java

```diff
@@ -167,70 +151,44 @@ public void onResponse(Call<LoginResponse> call, Response<LoginResponse> respons
                     PrefManager.putString(Constant.ACCESS_TOKEN, response.body().getAccessToken());
                     long validity = System.currentTimeMillis() + response.body().getValidSeconds() * 1000;
                     PrefManager.putLong(Constant.TOKEN_VALIDITY, validity);
-
                     realm.executeTransaction(new Realm.Transaction() {
                         @Override
                         public void execute(Realm realm) {
                             realm.deleteAll();
                             Log.d(TAG, "execute: all messages deleted");
                         }
                     });
-
-
                     Intent intent = new Intent(LoginActivity.this, MainActivity.class);
                     intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
                     intent.putExtra(getApplicationContext().getString(R.string.first_time), true);
                     startActivity(intent);
                     finish();
                 } else if(response.code() == 422) {
-                        AlertDialog.Builder builder = new AlertDialog.Builder(LoginActivity.this);
-                        builder.setTitle(R.string.password_invalid_title);
-                        builder.setMessage(R.string.password_invalid)
-                                .setCancelable(false)
-                                .setPositiveButton(getApplicationContext().getString(R.string.ok), null);
-                        AlertDialog alert = builder.create();
-                        alert.show();
-                        Button pbutton = alert.getButton(DialogInterface.BUTTON_POSITIVE);
-                        pbutton.setTextColor(Color.BLUE);
+                    alertTitle = getResources().getString(R.string.password_invalid_title);
+                    alertMessage = getResources().getString(R.string.password_invalid);
+                    alertNotSuccess(alertTitle, alertMessage);
                 } else {
-                        AlertDialog.Builder builder = new AlertDialog.Builder(LoginActivity.this);
-                        builder.setTitle(response.code() + getApplicationContext().getString(R.string.error));
-                        builder.setMessage(response.message())
-                                .setCancelable(false)
-                                .setPositiveButton(getApplicationContext().getString(R.string.ok), null);
-                        AlertDialog alert = builder.create();
-                        alert.show();
-                        Button pbutton = alert.getButton(DialogInterface.BUTTON_POSITIVE);
-                        pbutton.setTextColor(Color.BLUE);
+                    alertNotSuccess(response.code()+getResources().getString(R.string.error), response.message());
```

Comment:
@mayank408 everything is discussed with @chiragw15 [here](https://github.com/fossasia/susi_android/pull/684). I changed code accordingly.